src/main/java/com/endava/internship/service/UserServiceImpl.java

```diff
@@ -2,68 +2,90 @@
 
```

Comment:
could you please use this formatting for all lambda expressions, ex:
users.stream()
                .map(...
                .sorted(...
                .collect(Collectors.toList());

Reply:
resolved